commands/language.js

```diff
@@ -0,0 +1,40 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const {MessageActionRow, MessageEmbed, MessageSelectMenu, SelectMenuInteraction} = require('discord.js');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('language')
+        .setDescription('sets your preferred language'),
+    async execute(interaction) {
+        const row = new MessageActionRow()
+            .addComponents(
+                new MessageSelectMenu()
+                    .setCustomId('language')
+                    .setPlaceholder('Nothing selected')
+                    .addOptions([
+                        {
+                            label: 'English',
+                            value: 'en',
+                        },
+                        {
+                            label: 'Español',
+                            value: 'es',
+                        }, {
+                            label: 'Deutsch',
+                            value: 'de',
+                        }
+                    ]),
+            );
+
+        await interaction.reply({content: 'Please Select a language.', ephemeral: true, components: [row]});
+    },
+    async executeInteraction(/** MessageComponentInteraction */ interaction, userPrefs){
+        if (!interaction.isSelectMenu()) return;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
        if (!interaction.isSelectMenu()) {return;}
```